src/urllib3/util/ssl_.py

```diff
@@ -34,19 +35,39 @@ def _const_compare_digest_backport(a, b):
 
 _const_compare_digest = getattr(hmac, "compare_digest", _const_compare_digest_backport)
 
+
+def _is_ge_openssl_v1_1_1(
```

Comment:
This function derived from the suggestion by @tiran: https://github.com/urllib3/urllib3/issues/2090#issuecomment-733752400